cmd/main.go

```diff
@@ -134,29 +172,26 @@ func StartIPPoolController(mgr ctrl.Manager, commonService common.Service) {
 	}
 }
 
-func StartVPCController(mgr ctrl.Manager, commonService common.Service) {
+func StartVPCController() {
 	vpcReconciler := &vpccontroller.VPCReconciler{
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
 	}
-	if vpcService, err := vpc.InitializeVPC(commonService); err != nil {
-		log.Error(err, "failed to initialize vpc commonService", "controller", "VPC")
-		os.Exit(1)
-	} else {
-		vpcReconciler.Service = vpcService
-		commonctl.ServiceMediator.VPCService = vpcService
-	}
+
+	vpcReconciler.Service = vpcService
+
 	if err := vpcReconciler.Start(mgr); err != nil {
 		log.Error(err, "failed to create vpc controller", "controller", "VPC")
 		os.Exit(1)
 	}
 }
 
-func StartNamespaceController(mgr ctrl.Manager, commonService common.Service) {
+func StartNamespaceController() {
```

Comment:
https://github.com/vmware-tanzu/nsx-operator/pull/280 in this PR, we are tarting to move start controller into controller, so do we consider this method, or we still use PR 280 to do it?